Cargo.toml

```diff
@@ -16,5 +16,7 @@ license-file = "LICENSE"
 [workspace.dependencies]
 serde = { version = "1.0.193", features = ["derive", "rc"] }
 serde_json = "1.0.108"
+openssl = { version = "0.10.66", features = ["vendored"] }
+openssl-sys = { version = "0.9.103", features = ["vendored"] }
```

Comment:
I'm wondering if this is necessary (also in `crates/tbdex/Cargo.toml`)? I ask because it wasn't necessary in [this draft PR](https://github.com/TBD54566975/tbdex-rs/pull/91/files). I know for sure it's necessary specifically inside of `bindings/tbdex_uniffi/Cargo.toml` for the x86_64 macOS build to pass, but I wasn't aware of it being necessary here and inside of `crates/tbdex/Cargo.toml`.

Reply:
I've incorporated these deletions into the PR - thanks @KendallWeihe !